packages/@okta/vuepress-site/docs/guides/sign-your-own-saml-csr/sign-the-csr/index.md

```diff
@@ -2,7 +2,22 @@
 title: Sign the CSR
 ---
 
-Follow the third-party process that your company uses to sign the CSR. You can't move to step four until you complete the process because the signed CSR is required to continue.
+Follow the third-party Certificate Authority (CA) process that your company uses to sign the CSR. You can't proceed to the next step until your certificate is signed using this process.
+
+The following is a list of popular third-party CA providers that can be used to sign the CSR:
+ - Comodo SSL
+ - DigiCert
+ - GeoTrust
+ - GlobalSign
+ - GoDaddy
+ - Network Solutions
+ - RapidSSL
+ - SSL.com
+ - Thawte
+
+> **Note:** There is a cost associated with SSL certificates being signed by a third-party CA.
+
+The certificate authority you choose will provide instructions on how to upload the CSR you generated in the previous steps.
```

Comment:
The certificate authority that you choose provides instructions on how to upload the CSR that you generated in the previous steps.

Reply:
Nit: we mention CA above and certificate authority here. Should we define earlier that CA = certificate authority? Or, if not, mention here `certificate authority (CA)`

Reply:
I recommend using certificate authority (CA)